plugin/translate.vim

```diff
@@ -91,7 +121,10 @@ augroup translate
   autocmd bufenter * if (winnr("$") == 1 && s:is_trans_buf_open()) | q! | endif
 augroup END
 
-command! -nargs=? Translate call s:translate(<q-args>)
-command! -nargs=? -range TranslateVisual call s:translate_visual(<q-args>)
-command! -nargs=? -range TranslateReplace call s:translate_replace(<q-args>)
+
+" Commands {{{
```

Comment:
What is the purpose of this {{{ ?

Reply:
decoration 😄just kidding, it's folding markers

Reply:
Oh nice :))